applications/asset_tracker_v2/src/modules/data_module.c

```diff
@@ -964,29 +964,19 @@ static void agps_request_handle(struct nrf_modem_gnss_agps_data_frame *incoming_
 {
 	int err;
 
-	/* If the A-GPS library is enabled, it will handle the creation of the A-GPS request,
-	 * send it, and process the returned data.
-	 *
-	 * If CONFIG_NRF_CLOUD_MQTT is enabled, the nRF Cloud MQTT transport library will be used
-	 * to sent the request. This is handled internally in the A-GPS library.
-	 *
-	 * If CONFIG_AGPS_SRC_SUPL is enabled a SUPL session is started and A-GPS data is requested
-	 * from a configured SUPL server.
-	 *
-	 * If both CONFIG_NRF_CLOUD_MQTT and CONFIG_AGPS_SRC_SUPL are enabled, SUPL will take
-	 * precedence over nRF Cloud as it is selected as the A-GPS source. By default, nRF Cloud is
-	 * selected as the A-GPS source via the CONFIG_AGPS_SRC_NRF_CLOUD option.
+	/* If CONFIG_NRF_CLOUD_MQTT is enabled, the nRF Cloud MQTT transport library will be used
+	 * to send the request.
 	 */
-#if defined(CONFIG_AGPS) && (defined(CONFIG_NRF_CLOUD_MQTT) || defined(CONFIG_AGPS_SRC_SUPL))
-	err = agps_request_send(*incoming_request);
+#if defined(CONFIG_NRF_CLOUD_AGPS) && defined(CONFIG_NRF_CLOUD_MQTT)
```

Comment:
This effectively disables use of SUPL for A-GPS. This is not what it says in the commit.

Reply:
This only removes A-GPS library usage (along with SUPL support) from asset_tracker_v2. It doesn't remove the A-GPS library itself, which is still used by asset_tracker and the AGPS sample.